plugins/home/src/components/VisitList/VisitList.test.tsx

```diff
@@ -19,16 +19,9 @@ import { VisitList } from './VisitList';
 import { renderInTestApp } from '@backstage/test-utils';
 
 describe('<VisitList/>', () => {
-  it('renders with mandatory parameters', async () => {
-    const { getByText } = await renderInTestApp(
-      <VisitList title="My title" detailType="time-ago" />,
-    );
-    expect(getByText('My title')).toBeInTheDocument();
-  });
-
   it('renders skeleton when loading is true', async () => {
-    const { container } = await renderInTestApp(
-      <VisitList title="My title" detailType="time-ago" loading />,
+    const { container } = await render(
```

Comment:
Is there a reason you aren't using `renderInTestApp` anymore? It's generally the recommended method. That goes for all the cases this was changed in this file.

Reply:
Thanks @awanlin, I will take a look and fix it. I didn't change much in the test cases. When I had pushed the changes, all test cases passed. Some days later, it started showing merge conflicts, indicating some other code had been merged.

I will fix the test cases as soon as possible and ensure they pass successfully.

Reply:
Hi @awanlin , Changes are done and test cases are passing now. Please review.